platform/security/sts/security-sts-server/src/main/java/ddf/security/sts/AuthNStatementProvider.java

```diff
@@ -46,7 +46,10 @@ public AuthNStatementProvider() {
   @Override
   public AuthenticationStatementBean getStatement(TokenProviderParameters providerParameters) {
     AuthenticationStatementBean authBean = new AuthenticationStatementBean();
-    authBean.setSessionIndex(Integer.toString(secureRandom.nextInt()));
+    // "Use small positive integers (or reoccurring constants in a list) for the SessionIndex"
+    // Ref: sstc-saml-core-errata-2.0-wd-07, line 1140
+    int smallPositiveInteger = secureRandom.nextInt(Short.MAX_VALUE);
```

Comment:
So `nextInt` will return a number between [0, Short.MAX_VALUE) , since its inclusive of zero we could return zero. Not sure if we want to exclude it since it says small  **positive** integer

Reply:
Good point. I'll take care of that.